andela/hc-the-knights
# #154130050 heroku deployment

#### What does this PR do?
Deploy the app to Heroku

#### Description of Task to be completed
The deployed app should be able to run on Heroku

#### How can this be manually tested?
Check the Readme badge after a successful build
[hc-the-knights](https://hc-the-knights.herokuapp.com)

#### What are the relevant pivotal tracker stories?
[#154130050](https://www.pivotaltracker.com/story/show/154130050)
